boards/pimoroni/pico_plus2/doc/index.rst

```diff
@@ -0,0 +1,303 @@
+.. _rpi_pico:
+
+Raspberry Pi Pico
+#################
+
+Overview
+********
+
+The Raspberry Pi Pico and Pico W are small, low-cost, versatile boards from
+Raspberry Pi. They are equipped with an RP2040 SoC, an on-board LED,
+a USB connector, and an SWD interface. The Pico W additionally contains an
+Infineon CYW43439 2.4 GHz Wi-Fi/Bluetooth module. The USB bootloader allows the
+ability to flash without any adapter, in a drag-and-drop manner.
+It is also possible to flash and debug the boards with their SWD interface,
+using an external adapter.
+
+Hardware
+********
+- Dual core Arm Cortex-M0+ processor running up to 133MHz
+- 264KB on-chip SRAM
+- 2MB on-board QSPI flash with XIP capabilities
+- 26 GPIO pins
+- 3 Analog inputs
+- 2 UART peripherals
+- 2 SPI controllers
+- 2 I2C controllers
+- 16 PWM channels
+- USB 1.1 controller (host/device)
+- 8 Programmable I/O (PIO) for custom peripherals
+- On-board LED
+- 1 Watchdog timer peripheral
+- Infineon CYW43439 2.4 GHz Wi-Fi chip (Pico W only)
+
+
+.. figure:: img/rpi_pico.jpg
+     :align: center
+     :alt: Raspberry Pi Pico
+
+
+.. figure:: img/rpi_pico_w.jpg
+     :align: center
+     :alt: Raspberry Pi Pico W
+
+     Raspberry Pi Pico (above) and Pico W (below)
+     (Images courtesy of Raspberry Pi)
+
+Supported Features
+==================
+
+The rpi_pico board configuration supports the following
```

Comment:
board name as used by west with \`\` around it and board target, not configuration